lib/definitions/errors.js

```diff
@@ -26,19 +26,20 @@ Please make sure to add the \`repositoryUrl\` to the [semantic-release configura
       'docs/usage/configuration.md'
     )}).`,
   }),
-  EGITNOPERMISSION: ({options}) => ({
-    message: 'The push permission to the Git repository is required.',
+  EEGITNOPERMISSION: ({options}) => ({
```

Comment:
Please do not change the message identifier.

Reply:
Oh, I am terribly sorry, had a mishap with vim editing.